<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>1000</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="hlMain">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="Line" name="lLeft">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="br::GalleryViewer" name="gvTarget" native="true"/>
      </item>
      <item>
       <widget class="Line" name="lCenter">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="br::GalleryViewer" name="gvQuery" native="true"/>
      </item>
      <item>
       <widget class="Line" name="lRight">
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="br::Tail" name="tTail" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="br::Help" name="hHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="hHelp"/>
  </widget>
  <widget class="QToolBar" name="tbAlgorithm">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="br::View" name="vView">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>br::Help</class>
   <extends>QMenu</extends>
   <header>openbr-gui/help.h</header>
  </customwidget>
  <customwidget>
   <class>br::Tail</class>
   <extends>QWidget</extends>
   <header>openbr-gui/tail.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>br::GalleryViewer</class>
   <extends>QWidget</extends>
   <header>openbr-gui/galleryviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>br::View</class>
   <extends>QToolBar</extends>
   <header>openbr-gui/view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
